tests/test_block_validation.py

```diff
@@ -1,111 +1,130 @@
 from Crypto.PublicKey import RSA
 import pytest
 import os
+import shutil
+from time import time
 
 from blockchain.block import Block, create_initial_block
 from blockchain.transaction import *
 from blockchain.helper.block_validator import validate_block
 from blockchain.config import CONFIG
 
+GENESIS = None
+PUBLIC_KEY = None
+PRIVATE_KEY = None
 
-PUBLIC_KEY = RSA.import_key(open("tests" + os.sep + "testkey_pub.bin", "rb").read())
-PRIVATE_KEY = RSA.import_key(open("tests" + os.sep + "testkey_priv.bin", "rb").read())
 
+def setup_module(module):
+    # blockchain files folder will be deleted by create_initial_block
+    global GENESIS, PUBLIC_KEY, PRIVATE_KEY
+    GENESIS = create_initial_block()
```

Comment:
Does this overwrite the prod genesis block? (Why not?)

Reply:
It doesn't overwrite the prod block. In the config file, I detect, whether the script was called by pytest (which is actually not a nice way, but I had no better idea). Based on that, I set the blockchain files folder to `tests/blockchain_files` or  `blockchain/blockchain_files`. When creating the genesis block, it retrieves the path from config.